serverless.yml

```diff
@@ -27,11 +33,20 @@ custom:
     subnetIds: ${ssm:/iow/aws/vpc/${self:provider.stage}/subnetIds~split}
   db:
     connectInfo: ${ssm:/aws/reference/secretsmanager/NWCAPTURE-DB-${self:provider.stage}~true}
+  esLogs:
+    endpoint: ${ssm:/iow/aws/vpc/${self:provider.stage}/elasticsearchEndpoint}
+    index: dvstat
```

Comment:
Question more for my own understanding. Will the user need to remember this, like what the index will be for the other Lambda functions? If so, the convention could be to use the whole name: aqts-capture-dvstat-transform
not necessarily better, always a balance between verbosity and being easy understand.

Reply:
That's a good idea. 

Kibana also tells the user the available indices, but I do like having a convention for this.